build.sbt

```diff
@@ -12,10 +12,7 @@ lazy val microservice = (project in file("."))
   .enablePlugins(play.sbt.PlayScala, SbtAutoBuildPlugin, SbtDistributablesPlugin)
   .settings(commonSettings: _*)
   .settings(
-    Test / unmanagedSourceDirectories := Seq(
-      (Test / baseDirectory).value / "test/unit",
-      (Test / baseDirectory).value / "test/utils"
-    ),
+    Test / unmanagedSourceDirectories := Seq((Test / baseDirectory).value / "test/unit", (Test / baseDirectory).value / "test/utils"),
```

Comment:
Do not know if this change is due to some auto reformatting, but honestly it was more readable as it was before

Reply:
It is auto formatting, and I agree.

Reply:
It's weird btw to have auto-formatting on the build.sbt :raised_eyebrow: